packages/flow-lineage/src/components/f-lineage/draw/draw-nodes.ts

```diff
@@ -116,10 +116,16 @@ export default function drawNodes(params: DrawLineageParams) {
 					let childHeight = d.childrenYMax - (d.y + nodeSize.height);
 
 					// finding all nodes below children
-					const nodesToUpdate = lineage.nodes.filter(
+					let nodesToUpdate = lineage.nodes.filter(
```

Comment:
Why do extra work here? If it's vertical we throw away the previous work

Reply:
Code updated